.phpunit.result.cache

```diff
@@ -1 +1 @@
-{"version":1,"defects":[],"times":{"PrivacyIDEATest::testGetAuthToken":0.08,"PrivacyIDEATest::testEnrollToken":0.118,"PrivacyIDEATest::testPollTransaction":0.052,"PrivacyIDEATest::testTriggerChallenge":0.094,"PrivacyIDEATest::testValidateCheck":0.075}}
\ No newline at end of file
+{"version":1,"defects":{"PrivacyIDEATest::testTriggerChallenge":4,"PrivacyIDEATest::testValidateCheck":3},"times":{"PrivacyIDEATest::testGetAuthToken":0.349,"PrivacyIDEATest::testEnrollToken":0.596,"PrivacyIDEATest::testPollTransaction":0.202,"PrivacyIDEATest::testTriggerChallenge":0.462,"PrivacyIDEATest::testValidateCheck":0.491}}
```

Comment:
this file can be removed from git

Reply:
ok.

Reply:
phpunit.result.cache added to .gitignore and commited

Reply:
Also remove it from the PR please